app/models/student_loans/eligibility.rb

```diff
@@ -51,6 +51,7 @@ class Eligibility < ApplicationRecord
     validates :mostly_performed_leadership_duties, on: [:"mostly-performed-leadership-duties", :submit], inclusion: {in: [true, false], message: "Select yes if you spent more than half your working hours on leadership duties"}, if: :had_leadership_position?
     validates :student_loan_repayment_amount, on: [:"student-loan-amount", :submit], presence: {message: "Enter your student loan repayment amount"}
     validates_numericality_of :student_loan_repayment_amount, message: "Enter a valid monetary amount", allow_nil: true, greater_than: 0, less_than_or_equal_to: 99999
+    validates :student_loan_repayment_amount, on: :amendment, numericality: {less_than: 5000, message: "Enter an amount below £5,000"}
```

Comment:
I think it may be useful to have the error highlighted against the student loan input as well. I've included an example of how this is done for teacher reference numbers 

![image](https://user-images.githubusercontent.com/13239597/77074048-1b536200-69e8-11ea-8415-3a87063bddcc.png)

Reply:
Yeah good catch `errors_tag` wasn't displaying the error correctly because it's a nested attribute. I'll push up a fix